security/pkg/nodeagent/vm/citadelagent_v2_test.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vm
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"istio.io/istio/pkg/log"
+	"istio.io/istio/security/pkg/caclient"
+	providerMock "istio.io/istio/security/pkg/nodeagent/caclient/providers/mock"
+	"istio.io/istio/security/pkg/platform"
+	platformMock "istio.io/istio/security/pkg/platform/mock"
+	"istio.io/istio/security/pkg/util"
+	utilMock "istio.io/istio/security/pkg/util/mock"
+)
+
+func Test(t *testing.T) {
+	generalConfig := Config{
+		CAClientConfig: caclient.Config{
+			CAAddress:                 "ca_addr",
+			Org:                       "Google Inc.",
+			RSAKeySize:                512,
+			Env:                       platform.GcpVM,
+			CSRInitialRetrialInterval: time.Millisecond,
+			CSRMaxRetries:             3,
+			CSRGracePeriodPercentage:  50,
+			RootCertFile:              "ca_file",
+			KeyFile:                   "pkey",
+			CertChainFile:             "cert_file",
+		},
+		LoggingOptions: log.DefaultOptions(),
+	}
+	defer func() {
+		_ = os.Remove(generalConfig.CAClientConfig.RootCertFile)
```

Comment:
no need to do ` _ = ...`. if you are not consuming the return values?

Reply:
Removed.